airflow/www/utils.py

```diff
@@ -253,6 +253,7 @@ def nobr(attr):
     return nobr
 
 
+# pylint: disable=invalid-name
 def datetime_f(attr_name):
     """Returns a formatted string with HTML for given DataTime"""
     def dt(attr):
```

Comment:
```suggestion
    def dt(attr):  # pylint: disable=invalid-name
```
Should this do? I prefer inline comment as it's easier to understand where exactly is the problem

Reply:
Agree.